aws/routes.go

```diff
@@ -27,13 +27,21 @@ import (
 	"github.com/trackit/trackit2/users"
 )
 
+var (
+	// QueryArgAwsAccounts allows to get the AWS Account IDs in the URL Parameters
+	// with routes.RequiredQueryArgs. This AWS Account IDs will be a slice of Uint
+	// stored in the routes.Arguments map with itself for key.
+	QueryArgAwsAccounts = routes.QueryArg{"aa", "AWS Account IDs", routes.QueryArgUintSlice{}, false}
```

Comment:
Why don’t you put this definition next to that of `QueryArgAwsAccount` ? Please keep similar definitions in the same place.